src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+       >
+      </a>
+      <nav class="nav_panel">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="link is-active" href="./some_link">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="link" href="./some_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a class="link" href="./some_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="link"
+              href="./some_link"
+              data-qa="hover">
+                Laptops & computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a class="link" href="./some_link">Gadgets</a>
+          </li>
+          <li class="nav_item">
+            <a class="link" href="./some_link">Tablets</a>
+          </li>
+          <li class="nav_item">
+            <a class="link" href="./some_link">Photo</a>
```

Comment:
Replace the href value with '#' instead of './some_link' for demonstration purposes.